src/angular-app/bellows/apps/userprofile/user-profile-app.component.html

```diff
@@ -105,6 +110,66 @@ <h5>Tell us about yourself...</h5>
+                                </div>
+                            </div>
+                            <div ng-show="$ctrl.finishedLoadingOwnedProjects">
+                                <div class="card-body" ng-if="!$ctrl.ownsAProject">
+                                    <div class="form-group row">
+                                        <label class="col-12" for="deletebox">Confirm deletion by typing DELETE into the box below</label>
+                                        <div class="col-md-4">
+                                            <input data-ng-model="deleteBoxText" class="form-control" type="text" id="deletebox">
+                                        </div>
+                                    </div>
+                                    <div class="form-group row">
+                                        <div class="col-12">
+                                            <button id="deleteAccountBtn" class="btn btn-danger"
+                                                    data-ng-click="$ctrl.deleteOwnAccount()">
+                                                    <!--Need to change this ng-click!!-->
+                                                <i class="fa fa-trash"></i> Delete my account</button>
+                                        </div>
+                                    </div>
+                                    <div class="text-error">Deleting your account is a permanent action and cannot be undone. Your account will be deleted, and you will be removed from all projects you are a member of. You will no longer be able to log in.</div>
+                                </div>
+                                <div class="card-body" ng-if="$ctrl.ownsAProject">
+                                    <h5>You are a project owner.</h5>
+                                    <div class="form-group">
+                                        First, transfer
+                                    </div>
+                                    <listview items="$ctrl.ownedProjects">
+                                        <div data-ng-repeat="project in $ctrl.ownedProjects">
+                                            <div class="row text-center">
+                                                <div class="col-5">
+                                                    <a data-ng-href="/app/lexicon/{{project.id}}">
+                                                        <span>{{project.projectName}}</span>
+                                                    </a>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </listview>
+                                    <br>
+                                    <div class="form-group">
+                                        to a new owner or delete. Then, you will be able to delete your account.
```

Comment:
I agree, thanks Tim.

Reply:
👍 thank you. I struggled with writing this because the user might only own one project, so didn't know whether to use singular or plural for projects. I like your wording though; accepted.